devel/include/client/current_time.h

```diff
@@ -0,0 +1,227 @@
+// Generated by gencpp from file client/current_time.msg
+// DO NOT EDIT!
+
+
+#ifndef CLIENT_MESSAGE_CURRENT_TIME_H
+#define CLIENT_MESSAGE_CURRENT_TIME_H
+
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include <ros/types.h>
+#include <ros/serialization.h>
+#include <ros/builtin_message_traits.h>
+#include <ros/message_operations.h>
+
+
+namespace client {
+    template<class ContainerAllocator>
+    struct current_time_ {
+        typedef current_time_<ContainerAllocator> Type;
+
+        current_time_()
+                : name(), year(0), month(0), day(0), hour(0), minute(0), second(0) {
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]